src/status_im/ui/screens/network_settings/events.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im.ui.screens.network-settings.events
+  (:require [re-frame.core :as re-frame]
+            [taoensso.timbre :as log]
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.fleet.core :as fleet-core]
+            [status-im.utils.handlers :as handlers]
+            [status-im.utils.fx :as fx]))
+
+(defn- show-popup [title msg]
+  {:utils/show-popup [title msg nil]})
```

Comment:
show-popup now takes a map of arguments once my PR is in develop, also don't use a function that just return a map the map can be used directly especially if there is no logic